diagnostic.md

```diff
@@ -57,13 +65,15 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('#li').hide();
```

Comment:
This has the same issue as the previous answers that have `#li` as a selector.  One possible way is `$('li').parent().parent().hide();`.